### DISCUSSION TITLE
Instructions for TreeSitter configuration

### DISCUSSION THREAD

**Discussion by bdemick** [GumTreeDiff/gumtree#274] [Q&A] [answered]
It looks like it's possible to select TreeSitter as a TreeGenerator, but I can't seem to properly configure my system (macOS) to make this work. I also don't see any instructions in the wiki for configuration. When I try to use the `gen.treesitter.CTreeSitterTreeGenerator` API, I get an IOException because `tree-sitter-parser.py` doesn't exist. 

I have the tree-sitter-cli and Python bindings installed, and the native parser libraries that I want to use, but I'm not finding the missing `tree-sitter-parser.py` file that the CTreeSitterTreeGenerator API seems to be looking for.

**Comment by jrfaller** [comment, marked as answer]
Hi!

You need to install https://github.com/GumTreeDiff/tree-sitter-parser

Cheers!

**Reply by bdemick** [reply]
Totally missed that, thank you so much!
